tools/base/fpgainfo/bmcdata.h

```diff
@@ -36,15 +36,17 @@
 extern "C" {
 #endif
 
-typedef enum metrics_inquiry { ALL, POWER, THERMAL, PERF } metrics_inquiry;
+typedef enum metrics_inquiry { FPGA_ALL, FPGA_POWER, FPGA_THERMAL, FPGA_PERF } metrics_inquiry;
 
 fpga_result get_metrics(fpga_token token, 
```

Comment:
`get_metrics` and `print_metrics` seems a little too generic. Should we preface it with something like `bmc_` or `fpga_bmc_`?

Reply:
Power, thermal, Perf are generic,  BMC is not applicable for MCP platform